addons/account/models/account_move.py

```diff
@@ -4747,3 +4747,25 @@ def get_invoice_localisation_fields_required_to_invoice(self, country_id):
         :return: an array of ir.model.fields for which the user should provide values.
         """
         return []
+
+    def action_open_moves(self, res_ids=None, title=None):
```

Comment:
should we be adding these methods on models in account if it's only used by l10n_it_edi?
also they don't seem consistent, some go to the form if there is only 1 id
does it need the `view_mode` if the `views` are specified?

Reply:
> should we be adding these methods on models in account if it's only used by l10n_it_edi?
You're missing context: `saf-t` and `intrastat` use the old `file_generation_errors` widget and have those actions.
That widget will be replaced by this one in a following PR.

> they don't seem consistent, some go to the form if there is only 1 id
Only the tax action is different, can fix. Settings doesn't count of course.

> does it need the view_mode if the views are specified?
Probably not, investigating.

Reply:
Removed, factorized.